### DISCUSSION TITLE
Wagtail 3.0 released

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#8551] [Announcements]
I'm delighted to announce that Wagtail 3.0 is now out!

New features in this release include...

* **A brand new design**. The admin interface has been updated with a new sidebar, font stack, tab design and page editor, with improvements to performance and accessibility.
* **StreamField rich text block splitting** - rich text blocks can now be split to insert additional block content within existing spans of text.
* **Simplified panel definitions** - no need to remember to use `StreamFieldPanel`, `PageChooserPanel` and so on. Additionally, fields can now be designated as only editable by users with specified permissions.
* **Page descriptions** - provide additional information to help editors select the right page type.
* **Image duplicate detection** - warn editors if they upload an image that already exists in the library.
* **Image rendition prefetching** - improved performance for pages with large sets of images.

And much more - see the full release notes at https://docs.wagtail.org/en/stable/releases/3.0.html.

Be sure to check the [upgrade considerations](https://docs.wagtail.org/en/stable/releases/3.0.html#upgrade-considerations-changes-affecting-all-projects), as the changes in this release are more extensive than usual. For most sites, upgrading should still be a smooth process, assisted by the `wagtail updatemodulepaths` command. Remember to check for updates to any add-on packages you're using. For general advice on upgrading, please see: https://docs.wagtail.org/en/stable/releases/upgrading.html

Enjoy the new release!

**Comment by lb-** [comment]
Awesome! Thanks for the incredible work on this release.
Special shout out to @thibaudcolas and @stevedya for the bulk of the UI work (plus many others behind the scenes) and @gasman for the huge panels rework.
